features/consent/src/main/java/in/projecteka/jataayu/consent/ui/fragment/ConsentListFragment.kt

```diff
@@ -238,21 +227,15 @@ class GrantedFragment : BaseFragment(), AdapterView.OnItemSelectedListener,
             intent.putExtra(CONSENT_FLOW, ConsentFlow.GRANTED_CONSENTS.ordinal)
             startActivity(intent)
             EventBus.getDefault().postSticky(iDataBindingModel.id)
-
-            if (!EventBus.getDefault().isRegistered(this)) {
-                EventBus.getDefault().register(this)
             }
         }
-    }
 
-    @Subscribe
-    fun onEventReceived(messageEventType: MessageEventType) {
-        if (messageEventType == MessageEventType.CONSENT_REVOKED) {
-            viewModel.getConsents()
-            unregisterEventBus()
-        } else if (messageEventType == MessageEventType.USER_VERIFIED) {
-            viewModel.getGrantedConsentDetails(consentToRevoke.id)
-            unregisterEventBus()
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == 701) {
```

Comment:
use the constant from companion object. makes it easier to keep track of values being used.